test/testsuites/operatorhubtests.go

```diff
@@ -439,8 +440,8 @@ func checkCatsrcIsDeleted(name, namespace string) error {
 	err := wait.Poll(time.Second*5, time.Minute*1, func() (done bool, err error) {
 		def := &olm.CatalogSource{}
 		err = client.Get(context.TODO(), types.NamespacedName{Name: name, Namespace: namespace}, def)
-		if !errors.IsNotFound(err) || def.Name != "" {
-			return false, fmt.Errorf("default CatalogSource still present on the cluster")
+		if errors.IsAlreadyExists(err) || def.Name != "" {
```

Comment:
When would GET return 409?

Reply:
Good catch - I think I was looking to reverse the double negative for easier readability, but I'm just going to revert that change entirely as I'm not 100% convinced that the current check is correct.